docs/user/support.rst

```diff
@@ -1,53 +1,61 @@
 Site support
 ============
 
-Usage questions
----------------
+Read the Docs offers support for projects on our :doc:`Read the Docs for Business </commercial/index>` and |org_brand| platforms.
+We're happy to assist with any questions or problems you have using either of our platforms.
 
-If you have questions about how to use Read the Docs, or have an issue that
-isn't related to a bug, `Stack Overflow`_ is the best place to ask.  Tag
-questions with ``read-the-docs`` so other folks can find them easily.
+.. note::
+   Read the Docs does not offer support for questions or problems with documentation tools or content.
+   If you have a question or problem using a particular documentation tool,
+   you can refer to `community provided resources <Community provided resources>`_ instead.
 
-Good questions for Stack Overflow would be:
+Some examples of requests that we support are:
 
-* "What is the best way to structure the table of contents across a project?"
-* "How do I structure translations inside of my project for easiest contribution from users?"
-* "How do I use Sphinx to use SVG images in HTML output but PNG in PDF output?"
+- "How do I transfer ownership of a Read the Docs project to another maintainer?"
+- "Why are my project builds being cancelled automatically?"
+- "How do I manage my subscription?"
 
 You might also find the answers you are looking for in our :doc:`documentation guides </guides/index>`.
-These provide step-by-step solutions to common user requirements.
+These provide step-by-step solutions to common user requests.
+
+.. tabs::
 
-User support
-------------
+   .. tab:: |com_brand|
 
-If you need a specific request for your project or account,
-like more resources, change of the project's slug or username.
+        Please fill out the form at https://readthedocs.com/support/.
 
-.. tabs::
+        Our team responds to support requests during our normal business hours,
+        Monday through Friday, 8:00am to 5:00pm Pacific time.
+        We will reply within 2 business days or earlier for most plans.
 
    .. tab:: |org_brand|
 
         Please fill out the form at https://readthedocs.org/support/,
         and we will reply as soon as possible.
 
-   .. tab:: |com_brand|
+Community provided resources
```

Comment:
I was not happy with reusing the word "community" here, as we reference Read the Docs Community above and it's a little confusing with the overlap in terms.

I didn't find the previous heading, "Usage support", a good fit though. That is, usage with Read the Docs is still something we'd support, usage with Sphinx is not.

Reply:
Perhaps `External resources`?

Reply:
That works great, I've updated the section and link